fastlane/Pluginfile

```diff
@@ -6,6 +6,6 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 4.1'
 end
 
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit', branch: 'trunk' # Version 4.1 + fixes for promo screenshots helper (pending the next toolkit release)
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 5.2'
```

Comment:
FYI, if you're interested, once this PR lands, thanks to 5.2 you'll easily be able to open a PR to start using custom messages for Beta & Release builds on Buildkite by amending [this lane (beta)](https://github.com/woocommerce/woocommerce-ios/blob/trunk/fastlane/Fastfile#L395-L397) and [that one (release)](https://github.com/woocommerce/woocommerce-ios/blob/trunk/fastlane/Fastfile#L408-L410) to just add a `message:` parameter to those calls (see [here](https://github.com/wordpress-mobile/WordPress-Android/pull/17013/files#diff-4f4ec77529cea8e5a9f3f871c7e8e4d94bce0e51abff5634400b16b916973e15) for inspiration) 🙂